### ISSUE TITLE
remove social auth tables

### ISSUE THREAD

**Issue by willkg** [mozilla/standup#305]
When removing python-social-auth, I didn't remove the tables that it uses. So we're in this situation where there are foreign keys from those tables, but Django doesn't know about them so shenanigans.

We need to remove those tables because they break the Django admin.

**Comment by willkg** [comment]
I tried to do a migration but that created a lot of other issues.

So I did a backup and then did it manually in -prod:

```
~/mozilla/standup> heroku pg:psql --app standups
--> Connecting to xxx
psql (9.5.5, server 9.3.14)
SSL connection (protocol: TLSv1.2, cipher: DHE-RSA-AES256-GCM-SHA384, bits: 256, compression: off)
Type "help" for help.

standups::RED=> \d
standups::RED=> \d
standups::RED=> drop table if exists social_auth_association cascade;
DROP TABLE
standups::RED=> drop table if exists social_auth_code cascade;
DROP TABLE
standups::RED=> drop table if exists social_auth_nonce cascade;
DROP TABLE
standups::RED=> drop table if exists social_auth_usersocialauth cascade;
DROP TABLE
standups::RED=> \q
```

We still need to fix -stage and we should probably fix PR #306 so that it does things correctly.

**Comment by willkg** [comment]
-stage admin seems fine. I think either @pmac or I fixed -stage already or we refreshed -stage or something. Anyhow, this is a non-issue now, so closing it out.
